manifests/controller/cdi-controller-deployment.yaml

```diff
@@ -1,3 +1,36 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: cdi-sa
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cdi
+rules:
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
+  verbs: ["get", "list", "watch", "create", "update", "patch"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch", "create"]
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: cdi-sa
+roleRef:
+  kind: ClusterRole
+  name: cdi
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: cdi-sa
+    namespace: default
```

Comment:
Using the _default_ ns was one of the concerns in issue #136. Should this be left blank?

Reply:
it can't, that's the problem, we have to pass in a NS - in ansible template we can get around it by passing in `{{ cdi_namespace }}` but there is not live var substitution, so people need to be aware and change that to the proper NS I guess